test/integration/puppetlabs/services/master/environment_modules_int_test.clj

```diff
@@ -0,0 +1,78 @@
+(ns puppetlabs.services.master.environment-modules-int-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.http.client.sync :as http-client]
+            [puppetlabs.kitchensink.core :as ks]
+            [puppetlabs.puppetserver.bootstrap-testutils :as bootstrap]
+            [puppetlabs.trapperkeeper.testutils.logging :as logutils]
+            [puppetlabs.puppetserver.testutils :as testutils]
+            [cheshire.core :as cheshire]
+            [me.raynes.fs :as fs]
+            [clojure.tools.logging :as log]
+            [puppetlabs.services.jruby.jruby-puppet-testutils :as jruby-puppet-testutils])
+  (:import (com.puppetlabs.puppetserver JRubyPuppetResponse JRubyPuppet)
+           (java.util HashMap)))
+
+(def test-resources-dir
+  "./dev-resources/puppetlabs/services/master/environment_modules_int_test")
+
+(use-fixtures :once
+              (testutils/with-puppet-conf
+               (fs/file test-resources-dir "puppet.conf")))
+
+(defn get-env-modules
+  ([env-name]
+   (get-env-modules env-name nil))
+  ([env-name if-none-match]
```

Comment:
Looks like you could get rid of the overload which takes an `if-none-match` argument since the deftest doesn't use it.